File: back/models/place.py
class Place:
    def __init__(self, data, category, price=0, score=0):
        self.name = data.get('name')
        self.address = ','.join(data.get('location').get('formattedAddress'))
        self.lat = data.get('location').get('lat')
        self.lng = data.get('location').get('lng')
        self.category = category
        self.price = price
        self.score = score

File: web/lib/src/chat/models/place.dart
import 'dart:convert';

class Place {

  String name;
  String address;
  String category;

  int price;
  int score;

  Place({this.name, this.address, this.category, this.price, this.score});

  factory Place.fromJson(Map<String, dynamic> data){
    return Place(
      name: data['name'],
      address: data['address'],
      category: data['category'],
      price: data['price'],
      score: data['score']
    );
  }
}